test-worker/.gitignore

```diff
@@ -0,0 +1 @@
+package-lock.json
```

Comment:
think we can remove this `.gitignore` file, and use the  `.gitignore` at root instead ?

Reply:
No, we only want to gitignore the `package-lock.json` file here in the `test-worker/` directory (to make this test worker's npm always use latest versions), but not in the root (our sdk package.json).

Reply:
ok, so should we add `build` in this `.gitignore` ?